PublicMapping/districtbuilder
# Style map message and add zoom functionality

## Overview
This PR styles the "zoom in to work with blocks" message to pop more against its background. It also turns it into a button so that a user can quickly get to the correct zoom level to work with blocks.

### Checklist


### Demo
<img width="1053" alt="Screen Shot 2020-09-14 at 1 17 42 PM" src="https://user-images.githubusercontent.com/5672295/93107083-07dc3780-f6a9-11ea-9083-bd7c48bbf18e.png">

![districtbuilder_zoom_button](https://user-images.githubusercontent.com/5672295/93107480-918c0500-f6a9-11ea-9114-a30368a66f61.gif)

### Notes
N/A

## Testing Instructions
- `git pull`
- Open up a map. Zoom in close enough that the "Blocks" button is enabled. Click "Blocks" to begin editing at the block level.
- Now use the - arrow to zoom out.
- You should see the map message that says "zoom in to work with blocks". It should have a hover state and display a pointer cursor.
- Click the button.
- The message should disappear and you should see blocks again.

Closes #372
